src/context/ChatContext.tsx

```diff
@@ -36,6 +36,7 @@ export type ChatContextValue<
     watchers?: { limit?: number; offset?: number },
     event?: React.SyntheticEvent,
   ) => void;
+  useImageFlagEmojisOnWindows?: boolean;
```

Comment:
since we give this a default value in Chat looks like it doesn't need to be optional

Reply:
might be a good idea to look at all these types actually, some might always exist

Reply:
Right, I figured we might be doing that because technically we don't really need to have a value for this to work. I thought maybe the other attributes were also optional for that reason, but I can change it if needed